### DISCUSSION TITLE
Changing attraction/vehicle names and descriptions

### DISCUSSION THREAD

**Discussion by NerdMan8428** [OpenRCT2/OpenRCT2#17511] [Q&A]
I am trying to add real-world manufacturers to ride type names and vehicle names, as well as adding some minor historical facts to ride descriptions, more for my entertainment and immersion than anything else. I also stream the game and my viewers have become interested in these changes-it becomes great fuel for conversations. I have gone to editing the language files and this seems to work well enough for the tracked attractions, but flat rides still appear to be unedited. Elsewhere in the JSON files for vehicle types, I edit these files just the same but no changes appear in game.

I have been looking around online for a few days now and have been unable to find much input on this issue. It should seem like these changes would be rather strait-forward to me so I am quite confused. Are these superficial changes possible, and how could I go about doing them? 

Thank you very much!

**Comment by X123M3-256** [comment]
Well, it's definitely possible, those strings are stored somewhere. But if you've edited both the language file and the JSON objects, I'm not sure what else you'd be missing. Maybe you edited the en-GB language file and your game is set to en-US (or vice versa)?

**Reply by NerdMan8428** [reply]
Thank you for responding so quickly!

Those were my thoughts as well, conceptually this should be very simple to do. I edited both the en-GB and en-US files, but the US variant only has strings that require some semantic changes between both English dialects, so it does not have all available strings as much as I am aware. In the JSON, I replaced both the GB and US strings to have the custom names and descriptions, neither of which appear in-game.

**Reply by duncanspumpkin** [reply]
It's probably because you are running from a different set of files than what you are editing.

**Reply by NerdMan8428** [reply]
I didn't think of that for some reason. I'm not certain, as the saves and installation all go to the same file. It is definately possible there might be some extra residuals somewhere. Since the problem was happening in more than one PC though, I have my doubts. Still, I will check it and let you know.

**Reply by NerdMan8428** [reply]
I did a clean install and that seemed to do the trick. There was no change in the file locations, and no duplicate locations, my only guess is that an older version of the game utilized a slightly different coding format. Thank you!
